src/main/java/org/testng/reporters/JUnitReportReporter.java

```diff
@@ -185,6 +178,50 @@ public void generateReport(List<XmlSuite> xmlSuites, List<ISuite> suites,
 
   }
 
+  private TestTag createIgnoredTestTagFor(ITestNGMethod method) {
```

Comment:
Ok time for getting some learning. Can you please let me know as to what sort of impact would it have if we translate this method into a static method. I get the basic idea that methods that don't interact with the internal states of a class are good candidates for being static methods, but I was curious to understand as to what are the drawbacks in leaving these kind of methods as instance methods rather than static methods. Any pointers around that @juherr  ?